src/Datadog.Trace.OpenTracing/OpenTracingSpan.cs

```diff
@@ -101,6 +101,9 @@ public ISpan SetTag(string key, string value)
                 case DatadogTags.ServiceName:
                     Span.ServiceName = value;
                     return this;
+                case DatadogTags.ServiceVersion:
+                    Span.SetTag(Tags.Version, value);
```

Comment:
Isn't this case handled as a custom tag in `Span.SetTag()` below? This switch is only for special tags that actually set `Span` properties.

Reply:
The goal is for the setting of the `service.version` tag to actually set the span's `version` tag, which this accomplishes. Though the way this is written, the `service.version` tag doesn't actually get set

Reply:
There's already a call to `Span.SetTag()` in line 115 below. If the tags is not one of the special tags caught in this `switch`, it will be set as a tag on the span there. I think we can remove this entire `case DatadogTags.ServiceVersion`.

Reply:
I'm intentionally casing on the string `DatadogTags.ServiceVersion` (or `service.version`) because I want to set a different tag name `Tags.Version` (or `version`) when this is called. Removing this case won't succeed

Reply:
Oh, I didn't realize `DatadogTags.ServiceVersion` and `Tags.Version` were different strings. Why is this?

Reply:
I'm not entirely sure why we standardized on this behavior, but you can see [this comment](https://github.com/DataDog/dd-trace-dotnet/pull/716#discussion_r420150135) for more info

Reply:
`version` is the tag we use internally across products.

`service.version` just happens to be an OT/OTel standard that we also want to be sure we have compatibility for, in case someone tries to use it.

Reply:
Going to mark as resolved for now, but feel free to re-open

Reply:
Makes sense. Thanks both for ELI5.